src/common/opencl.c

```diff
@@ -1152,57 +1154,52 @@ void dt_opencl_init(
 
   if(exclude_opencl)
   {
-    dt_print_nts(DT_DEBUG_OPENCL,
+    dt_print(DT_DEBUG_OPENCL,
                  "[opencl_init] opencl disabled due to explicit user request\n");
     goto finally;
   }
 
-  dt_print_nts(DT_DEBUG_OPENCL,
-               "[opencl_init] opencl related configuration options:\n");
-  dt_print_nts(DT_DEBUG_OPENCL,
-               "[opencl_init] opencl: %s\n", opencl_requested ? "ON" : "OFF" );
-  const char *str = dt_conf_get_string_const("opencl_scheduling_profile");
-  dt_print_nts(DT_DEBUG_OPENCL,
-               "[opencl_init] opencl_scheduling_profile: '%s'\n", str);
+  if(!opencl_requested)
+    dt_print(DT_DEBUG_OPENCL, "[opencl_init] opencl disabled via darktable preferences\n");
+
   // look for explicit definition of opencl_runtime library in preferences
   const char *library = dt_conf_get_string_const("opencl_library");
-  dt_print_nts(DT_DEBUG_OPENCL,
-               "[opencl_init] opencl_library: '%s'\n",
-               (strlen(library) == 0) ? "default path" : library);
-  str = dt_conf_get_string_const("opencl_device_priority");
-  dt_print_nts(DT_DEBUG_OPENCL,
-               "[opencl_init] opencl_device_priority: '%s'\n", str);
-  dt_print_nts(DT_DEBUG_OPENCL,
-               "[opencl_init] opencl_mandatory_timeout: %d\n",
-               dt_conf_get_int("opencl_mandatory_timeout"));
 
   // dynamically load opencl runtime and bind required symbols and test it
   cl->dlocl = dt_dlopencl_init(library);
   if(cl->dlocl == NULL)
   {
     logerror = "no working opencl library found";
```

Comment:
@jenshannoschwalm :
Just missed that, as `logerror` is displayed to user using `dt_control_log()` we need to have those strings translated. So we need `_(` and `)` around all those strings for all `logerror` assignment.

Reply:
Yes and sure. Also there have been non-translatable tool tips added for slider buttons. Will fix all in one PR.